src/server/node_manager.c

```diff
@@ -3040,7 +3042,12 @@ deallocate_job(mominfo_t *pmom, job *pjob)
 		free(new_exec_vnode);
 
 	}
-	set_scheduler_flag(SCH_SCHEDULE_TERM);
+	if (find_assoc_sched_pj(pjob, &psched))
+		set_scheduler_flag(SCH_SCHEDULE_TERM, psched);
+	else {
+		sprintf(log_buffer, "Unable to reach scheduler associated with job %s", pjob->ji_qs.ji_jobid);
```

Comment:
Please update it to "Unable to find scheduler associated with partition"

Reply:
Done